stock_mts_mto_rule/model/procurement.py

```diff
@@ -19,93 +19,102 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ###############################################################################
-from openerp import models, api
+from openerp import api, fields, models
```

Comment:
Oops, forgot that :)
I replaced all `openerp` by `odoo`, and removed the big license headers by OCA's one in all files.